Tests/AbstractTest.php

```diff
@@ -54,51 +54,51 @@ protected function createFilterConfiguration()
 
     protected function getMockCacheManager()
     {
-        return $this->getMock('Liip\ImagineBundle\Imagine\Cache\CacheManager', array(), array(), '', false);
+        return $this->getMockBuilder('Liip\ImagineBundle\Imagine\Cache\CacheManager')->getMock();
```

Comment:
So happy someone is taking the time to update the test suite to remove PHPUnit 5.x deprecations. I have a similar branch that removes all `getMock` methods in favor of `getMockBuilder`. I'll wait until this is merged to submit my PR for this.